tests/contrib/operators/test_emr_add_steps_operator.py

```diff
@@ -93,14 +109,22 @@ def test_render_template(self):
     def test_execute_returns_step_id(self):
         self.emr_client_mock.add_job_flow_steps.return_value = ADD_STEPS_SUCCESS_RETURN
 
-        # Mock out the emr_client creator
-        emr_session_mock = MagicMock()
-        emr_session_mock.client.return_value = self.emr_client_mock
-        self.boto3_session_mock = MagicMock(return_value=emr_session_mock)
```

Comment:
Why did you remove this block?

Reply:
Wasn't removed but moved to `setUp` since both the existing test case and the new test case use the mock.